tests/test_proxy_connect.py

```diff
@@ -101,7 +101,9 @@ def test_https_noconnect_auth_error(self):
         self._assert_got_response_code(407, l)
 
     def _assert_got_response_code(self, code, log):
+        print(log)
```

Comment:
Not exactly a leftover; stdout is hidden by default, but pytest shows it if tests fail. So by printing log it becomes easier to debug the issue in case of failures.